packages/back-end/src/jobs/updateExperimentResults.ts

```diff
@@ -321,7 +321,7 @@ async function sendSignificanceEmail(
             // this test variation has gone significant, and won
             experimentChanges.push(
               "The metric " +
-                getMetricById(m, experiment.organization) +
+                (await getMetricById(m, experiment.organization)) +
```

Comment:
Did you test this?  Don't you want to use the metric name in the string instead of the full object?